src/item/component/IconComponent.php

```diff
@@ -17,6 +17,7 @@ public function getName(): string {
 
 	public function getValue(): array {
 		return [
+			"texture" => $this->texture,
 			"textures" => [
 				"default" => $this->texture
 			]
```

Comment:
is this even necessary anymore?

Reply:
Apparently it is, it fixed it for me so maybe it will for others too 🤷‍♂️